<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menureciet">
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Roles</string>
    </property>
    <addaction name="separator"/>
    <addaction name="QA_Create_Role"/>
    <addaction name="QA_Modify_Role"/>
    <addaction name="QA_Assign_User"/>
   </widget>
   <widget class="QMenu" name="menuAuthorization">
    <property name="title">
     <string>Authorization</string>
    </property>
    <widget class="QMenu" name="menuUsers">
     <property name="title">
      <string>Users</string>
     </property>
     <addaction name="QA_Create_User"/>
     <addaction name="QA_Modify_User"/>
     <addaction name="QA_copy_User"/>
    </widget>
    <addaction name="menuUsers"/>
   </widget>
   <widget class="QMenu" name="menuMaster_Data">
    <property name="title">
     <string>Master Data</string>
    </property>
    <addaction name="QA_Branch"/>
   </widget>
   <widget class="QMenu" name="menuprivilages">
    <property name="title">
     <string>Privilages</string>
    </property>
    <addaction name="separator"/>
    <addaction name="QA_Create_Priv"/>
    <addaction name="QA_Modify_Priv"/>
   </widget>
   <widget class="QMenu" name="menuForm">
    <property name="title">
     <string>Form</string>
    </property>
    <addaction name="QA_Create_Form"/>
    <addaction name="QA_Modify_Form"/>
   </widget>
   <addaction name="menuAuthorization"/>
   <addaction name="menuMaster_Data"/>
   <addaction name="menureciet"/>
   <addaction name="menuprivilages"/>
   <addaction name="menuForm"/>
  </widget>
  <action name="actionsd">
   <property name="text">
    <string>sd</string>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>إضافه بيانات العملاء</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>إستعلام عن بيانات العملاء</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>إستعلام عن مشتريات عملاء بالقسم</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>إضافه بيانات العملاء(ملاحظات)</string>
   </property>
  </action>
  <action name="actionexist">
   <property name="text">
    <string>خروج</string>
   </property>
  </action>
  <action name="actionCreate_User">
   <property name="text">
    <string>Create User</string>
   </property>
  </action>
  <action name="actionModify_User">
   <property name="text">
    <string>Modify User</string>
   </property>
  </action>
  <action name="actionCopy_User">
   <property name="text">
    <string>Copy User</string>
   </property>
  </action>
  <action name="QA_Create_Role">
   <property name="text">
    <string>Create Role</string>
   </property>
  </action>
  <action name="QA_Modify_Role">
   <property name="text">
    <string>Modify Role</string>
   </property>
  </action>
  <action name="QA_Create_User">
   <property name="text">
    <string>Create User</string>
   </property>
  </action>
  <action name="QA_Modify_User">
   <property name="text">
    <string>Modify User</string>
   </property>
  </action>
  <action name="QA_copy_User">
   <property name="text">
    <string>Copy User</string>
   </property>
  </action>
  <action name="QA_Branch">
   <property name="text">
    <string>Branch</string>
   </property>
  </action>
  <action name="QA_Assign_User">
   <property name="text">
    <string>Assign User</string>
   </property>
  </action>
  <action name="QA_Create_Priv">
   <property name="text">
    <string>Create Privilage</string>
   </property>
  </action>
  <action name="QA_Modify_Priv">
   <property name="text">
    <string>Modify Privilage</string>
   </property>
  </action>
  <action name="QA_Create_Form">
   <property name="text">
    <string>Create Form</string>
   </property>
  </action>
  <action name="QA_Modify_Form">
   <property name="text">
    <string>Modify Form</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
